src/services/ProjectService.ts

```diff
@@ -23,6 +24,10 @@ import { BudgetService } from './BudgetService'
 import { ProposalInCreation } from './ProposalService'
 import { VestingService } from './VestingService'
 
+function newestVestingFirst(a: TransparencyVesting, b: TransparencyVesting): number {
+  return new Date(b.vesting_start_at).getTime() - new Date(a.vesting_start_at).getTime()
```

Comment:
If dates are the same maybe you could check `vesting_finish_at` to see which one finishes later (only for those weird cases where both begin at the same time)